### DISCUSSION TITLE
Connect failed to mqtt rc -5

### DISCUSSION THREAD

**Discussion by flogehring** [arendst/Tasmota#22634] [Ideas]
### PROBLEM DESCRIPTION

I just want to log this here, in case somebody else has the same problem.
After I've spent an hour trying to figure out why my device wont connect to my broker.
The host address may NOT contain `mqtt://`

This fails:
<img width="418" alt="image" src="https://github.com/user-attachments/assets/ed93a6ee-335b-4c3c-abd2-9e486c9e1924" />

This works
<img width="434" alt="image" src="https://github.com/user-attachments/assets/ead41bb8-e215-44dc-9b55-1c945d6bdacf" />

Feel free to close the issue
OR 
add a hint to the UI
OR
parse the input

**Comment by sfromis** [comment]
The hint is already in the UI. "Host" does not mean "URL".

**Comment by barbudor** [comment]
You are free to contribute a PR on the doc repo if you feel that what you read there was not clear enough
Thanks in advance for your contribution
